TROUBLESHOOTING.md

```diff
@@ -6,27 +6,13 @@ In the mean time, you may need to diagnose a particular issue and determine if i
 
 We're also actively developing debug tools, profilers, and improving our errors/warnings.  Please give us feedback if you have an issue with this infra, or an idea for an improvement.
 
-## File an Issue
-You should feel encouraged to [file a github issue](https://github.com/pytorch/torchdynamo/issues) and expect a timely response.
-
-Before filing an issue, read over the README.md, TROUBLESHOOTING.md, and search for similar issues.
-
-When filing an issue, please include
-- a minimal repro script
-- a description of the error
-- the expected behavior
-- your OS/python/pytorch version
-
 ## Graph Breaks
 Given a program like this,
 
 ```
-@torchdynamo.optimize(...):
-def some_fun(x):
+with torchdynamo.optimize(...):
```

Comment:
Don't we want to stick with the decorator syntax?

Reply:
Yeah good catch, I left this here from the previous version.